lib/thread.c

```diff
@@ -42,6 +41,22 @@ DEFINE_MTYPE_STATIC(LIB, THREAD_STATS, "Thread stats")
 
 DECLARE_LIST(thread_list, struct thread, threaditem)
 
+static int thread_timer_cmp(const struct thread *a, const struct thread *b)
+{
+	if (a->u.sands.tv_sec < b->u.sands.tv_sec)
```

Comment:
stupid question:

Would it be better to compare tv_usec first?  Since they are more likely to be different?

Reply:
Their difference is meaningless if `tv_sec` isn't the same...

Reply:
dmanit you are sorting while I was trying to return a quicker difference